diff --git a/Vagrantfile b/Vagrantfile
index b034c26..d09b76a 100644
--- a/Vagrantfile
+++ b/Vagrantfile
@@ -95,14 +95,14 @@ Vagrant.configure("2") do |config|
     provision.call(master, %w[rudder kubecfg controller-manager apiserver kubelet proxy scheduler], "master")
   end
 
-  (1..NUMBER_OF_MINIONS).each do |i|
-    config.vm.define "minion-#{i}" do |minion|
-      minion.vm.hostname = "minion-#{i}"
-      minion.vm.network :private_network, ip: MINION_IP_ADDRS[i-1]
+  #(1..NUMBER_OF_MINIONS).each do |i|
+  #  config.vm.define "minion-#{i}" do |minion|
+  #    minion.vm.hostname = "minion-#{i}"
+  #    minion.vm.network :private_network, ip: MINION_IP_ADDRS[i-1]
 
-      minion.vm.provision :file, :source => MINION_CONFIG_PATH, :destination => "/tmp/vagrantfile-user-data"
+  #    minion.vm.provision :file, :source => MINION_CONFIG_PATH, :destination => "/tmp/vagrantfile-user-data"
 
-      provision.call(minion, %w[rudder kubelet proxy scheduler], "minion-#{i}")
-    end
-  end
+      #provision.call(minion, %w[rudder kubelet proxy scheduler], "minion-#{i}")
+  #  end
+  #end
 end
